i18n/locales/vi-VN/dashboard/scripts.yml

```diff
@@ -5257,7 +5259,7 @@ vi:
               display_name: Nội dung khóa học bổ sung
           lessons:
             courseB_unplugged_MRF_2018:
-              name: courseB_unplugged_MRF_2018
+              name: courseB_unplugged_MRF\_2018
```

Comment:
The extra backslash issue! The translation doesn't have backslash https://crowdin.com/translate/codeorg/63/enus-vi#2667968. This extra character could be from the restoration process.